bigml/bigml.php

```diff
@@ -3297,10 +3321,11 @@ private function parseJsonResponse($response, $code, $headers) {
          $this->response["code"] = $code;
       } else {
 
-         $headers = explode("\n", $headers);
-         foreach($headers as $header) {
-            if (stripos($header, 'Location:') !== false) {
-               $cad = explode("Location:", $header);
+         $headers_arr = explode("\n", $headers);
+         $headers_arr = array_change_key_case($headers_arr, CASE_LOWER);
```

Comment:
This fixes my case with the lowercase "l", but then breaks for "Location" with a capital "L" because the $headers_arr has numeric keys and using array_change_key_case on that doesn't do anything to the values. 

A quick example of what I mean..
```
$headers = "Location: http://bigml.io/andromeda/prediction/6480929b923e164b7d3b9e98\r\n";
$headers_arr = explode("\n", $headers);
```
Outputs 
```
array:2 [
  0 => "Location: http://bigml.io/andromeda/prediction/6480929b923e164b7d3b9e98\r"
  1 => ""
]
```
So using the following on the resulting $headers_arr from above
```
$headers_arr = array_change_key_case($headers_arr, CASE_LOWER);
```
leaves the array unchanged
```
array:2 [
  0 => "Location: http://bigml.io/andromeda/prediction/6480929b923e164b7d3b9e98\r"
  1 => ""
]
```
thus will still error if the "Location" header has a capital "L".